Sources/ServerIO/DocumentIO.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+public class DocumentIO {
+   
+    public init(){}
+
+    public func writeText(requestBody: [String: String], path: String) {
```

Comment:
`requestBody` feels like the wrong name here, since `DocumentIO` shouldn't know anything about "requests".